# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../nvidia/module-install-24.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Do not call depmod (especially not without the currently building version
attached) as well as not injecting some device nodes ...

  - Rene Rebe <rene@exactcode.de>

--- NVIDIA-Linux-x86-1.0-7664-pkg1/usr/src/nv/Makefile.nvidia.orig	2005-05-25 20:20:05.000000000 +0200
+++ NVIDIA-Linux-x86-1.0-7664-pkg1/usr/src/nv/Makefile.nvidia	2005-06-13 10:00:48.000000000 +0200
@@ -116,9 +116,7 @@
 package-install: module suser-sanity-check rmmod-sanity-check 
 	mkdir -p $(MODULE_ROOT)/video && \
 	install -m 0664 -o root -g root nvidia.o $(MODULE_ROOT)/video/nvidia.o$(O) && \
-	PATH="$(PATH):/bin:/sbin" depmod -a && \
-		PATH="$(PATH):/bin:/sbin" modprobe nvidia && \
-	echo "nvidia.o installed successfully."; \
+	echo "nvidia.o installed successfully."
 
 $(VERSION_HDR): 
 	echo \#define NV_COMPILER \"`$(CC) -v 2>&1 | tail -n 1`\" > $@

